justfile

```diff
@@ -0,0 +1,29 @@
+build:
+	cargo build
+
+schema:
+  #!/usr/bin/env sh
+  for dir in contracts/*; do
+    if [ -d "$dir" ]; then
+      echo "Generating schema for $dir"
+      (cd "$dir" && cargo schema)
+    fi
+  done
```

Comment:
removing the raw files after they are generated here would be nice

Reply:
I added them to `.gitignore`, do you think that's enough?

Reply:
yeah that's fine